gl_context/src/common.rs

```diff
@@ -75,6 +76,27 @@ impl GLContextBuilder {
         self
     }
 
+    /// Sets the major version.
+    /// This is presently only relevant on Windows.
```

Comment:
We don't actually use `version_major` and `version_minor` anywhere yet (even on Windows). Is this something you're planning on changing?

Reply:
Whoops, totally overlooked the part where I actually implement the feature, haha. Fixed.